README.md

```diff
@@ -40,6 +40,7 @@ make
   Consul API queries to gather all information about each service. Health check
   information are available via `consul_health_service_status` as well, but
   only for services which have a health check configured. Defaults to true.
+* __`consul.health-status-code`:__  Returns a health status code for passing (0), warning (1) or critical (2) via `consul_health_service_status` and `consul_catalog_service_node_healthy`. Default health status code is passing (1) and critical/warning (0).
```

Comment:
It 's OK for me to have a status label with warning,critical,passing in the metric `consul_health_service_status`. 

Maybe I misunderstood your other comment that this is not going to work because of staleness issues:

"I'm not sure how you're not running into staleness issues with your current approach. I'd expect to experience a delay of 5 minutes (default staleness delta) until alerts successfully update after all possible state changes."